src/commands.py

```diff
@@ -122,10 +122,10 @@ def transfer(
         amount: Fraction, server: Server):
     """Transfer amount ¤ from source to destination with authorization
        from author on server"""
-    author = _get_account(author, server)
```

Comment:
These upper ones won't fail but change the scheme to be inconsistent and somewhat confusing imho. Either everything is `acc_` or (preferably) nothing is.